packages/dd-trace/src/priority_sampler.js

```diff
@@ -44,16 +45,19 @@ class PrioritySampler {
     this.update({})
   }
 
-  configure (env, { sampleRate, provenance = undefined, rateLimit = 100, rules = [] } = {}) {
+  configure (env, options={}) {
+    const { sampleRate, provenance = undefined, rateLimit = 100, rules = [] } = options
     this._env = env
     this._rules = this._normalizeRules(rules, sampleRate, rateLimit, provenance)
     this._limiter = new RateLimiter(rateLimit)
 
+    log.trace(() => `PrioritySampler.configure called on options=${JSON.stringify(options)}`)
```

Comment:
I wonder if this could be simplified. For example something like `log.trace('PrioritySampler.configure', () => ({ env, options }))` instead of having to write out the text each time.